migration/pom.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.axonframework</groupId>
+        <artifactId>axon</artifactId>
+        <version>4.7.2-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>migration</artifactId>
```

Comment:
Calling this `axon-migration`, with the package name being `migration`, is more inline with the other modules in Axon Framework.

Reply:
ArtifactId renamed & applied package name to recipes.